examples/data-objects/shared-text/src/client-ui-lib/controls/commandBox.tsx

```diff
@@ -62,26 +62,36 @@ export const CommandBox: React.FC<ICommandBoxProps> = (props: ICommandBoxProps)
     };
 
     const keydownHandler = (e: React.KeyboardEvent<HTMLInputElement>) => {
-        if (e.key === "Escape") {
-            dismissCommandBox();
-        } else if (e.key === "ArrowDown") {
-            if (arrowedCommand === undefined) {
-                if (matchingCommands.length > 0) {
-                    setArrowedCommand(0);
+        switch (e.key) {
+            case "Escape": {
+                dismissCommandBox();
+               break;
+            }
+            case "ArrowDown": {
+                if (arrowedCommand === undefined) {
+                    if (matchingCommands.length > 0) {
+                        setArrowedCommand(0);
+                    }
```

Comment:
nit

```suggestion
                if (arrowedCommand === undefined && matchingCommands.length > 0) {
                    setArrowedCommand(0);
                }
```

Reply:
This isn't comparable, since there is an `else` case immediately following the outer if.